preview/index.ts

```diff
@@ -0,0 +1,74 @@
+import { readFileSync, writeFileSync } from 'fs';
+import { NonEmptyString } from 'io-ts-types';
+import { getEmailNewsletters, rowToNewsletter } from '../src/jobs/newsletters';
+import {
+	EmailNewsletter,
+	EmailNewsletterType,
+} from '../src/models/newsletters';
+import { parseStringifiedCSV } from './csv';
+
+const USE_LIVE_DATA = !!process.env.USE_LIVE_DATA;
+const PREVIEW_OUTPUT_FILE_PATH = './preview/preview.json';
+const PREVIEW_DATA_SOURCE_FILE_PATH = './preview/sampleData.csv';
+
+const logFeedback = (
+	versionNumber: string,
+	newsletters: EmailNewsletter[],
+): void => {
+	console.log({ versionNumber });
+	console.log('INDEX\t MATCH\t NAME');
+	newsletters.forEach((newsletter, index) => {
+		console.log(
+			index,
+			'\t',
+			EmailNewsletterType.is(newsletter),
+			'\t',
+			newsletter.name,
+		);
+	});
+};
+
+const getEmailNewslettersFromLocalCsv = async (): Promise<
+	EmailNewsletter[]
+> => {
+	const csvData = await readFileSync(
+		PREVIEW_DATA_SOURCE_FILE_PATH,
+	).toString();
+	const cellsInRows = parseStringifiedCSV(csvData);
+	const unverifiedNewsletters = cellsInRows.slice(1).map(rowToNewsletter);
```

Comment:
sorry if i've missed this somewhere else, but what does "unverified newsletter" mean here?